packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListTable.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { alertFactory } from 'src/factories';
+import { formatDate } from 'src/utilities/formatDate';
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { AlertsListTable } from './AlertListTable';
+
+describe('Alert List Table test', () => {
+  it('should render the alert landing table ', async () => {
+    const { getByText } = renderWithTheme(
+      <AlertsListTable alerts={[]} isLoading={false} services={[]} />
+    );
+    expect(getByText('Alert Name')).toBeVisible();
+    expect(getByText('Service')).toBeVisible();
+    expect(getByText('Status')).toBeVisible();
+    expect(getByText('Last Modified')).toBeVisible();
+    expect(getByText('Created By')).toBeVisible();
+  });
+
+  it('should render the error message', async () => {
+    const { getByText } = renderWithTheme(
+      <AlertsListTable
+        alerts={[]}
+        error={[{ reason: 'Error in fetching the alerts' }]}
+        isLoading={false}
+        services={[]}
+      />
+    );
+    await waitFor(() => {
+      expect(getByText('Error in fetching the alerts')).toBeVisible();
+    });
```

Comment:
I don't think `await waitFor(...)` is needed. There is nothing async happening in the component to require this

```suggestion
    expect(getByText('Error in fetching the alerts')).toBeVisible();
```